Core/Systems/AltUseSystem.cs

```diff
@@ -1,33 +0,0 @@
-﻿namespace PathOfTerraria.Core.Systems;
-
-public class AltUseSystem : ModPlayer
-{
-	/// <summary>
-	/// This is the time between alt use activations / usage
-	/// </summary>
-	public int AltFunctionCooldown;
-	
-	/// <summary>
-	/// This is how long the alt use will be active for. This is meant for anything that has a duration.
-	/// </summary>
-	public int AltFunctionActiveTimer;
```

Comment:
Looks like the timer, and the comments were not brought over with the update

Reply:
Oh, this was not in the version of `main` I originally pulled. There's a ton I need to fix now, hold up